src/tests/src/libs/antares/study/wrappers/study-wrapper.h

```diff
@@ -0,0 +1,79 @@
+
+#include <study.h>
+
+namespace Antares
+{
+namespace Data
+{
+class Study;
+/*!
+** \brief Antares StudyWrapper
+*/
+
+using pStudy = std::shared_ptr<Study>;
+class StudyWrapper
+{
+
+public:
+    using Ptr = std::shared_ptr<StudyWrapper>;
+
+    explicit StudyWrapper(pStudy _pstudy):
+        pstudy(_pstudy){};
+    
+    //! \name Loading wrapper
+    //@{
+    //! Load a study from a folder
+    bool internalLoadFromFolder(const YString& path, const StudyLoadOptions& options){
+        return pstudy->internalLoadFromFolder(path, options);
+    }
+
+    //! Load the study header
+    bool internalLoadHeader(const YString& folder){
+        return pstudy->internalLoadHeader(folder);
+    }
+    //! Load all correlation matrices
+    bool internalLoadCorrelationMatrices(const StudyLoadOptions& options){
+        return pstudy->internalLoadCorrelationMatrices(options);
+    }
+    //! Load all binding constraints
+    bool internalLoadBindingConstraints(const StudyLoadOptions& options){
+        return pstudy->internalLoadBindingConstraints(options);
+    }
+    //! Load all set of areas and links
+    bool internalLoadSets(){
+        return pstudy->internalLoadSets();
+    }
+    //@}
+
+    bool initializeInternalData(const StudyLoadOptions& options){
+        return pstudy->initializeInternalData(options);
+    }
+    void initializeSetsData(){
+        pstudy->initializeSetsData();
+    }
```

Comment:
Added for tests, not tested though. Either remove from this [attorney class](https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Friendship_and_the_Attorney-Client), or test.